tests/test_add_new_openwrap_partner.py

```diff
@@ -514,7 +514,7 @@ def test_openwrap_creative_type_inapp_video(self, mock_input, mock_load_price_cs
     #check roadblock type
     self.assertEqual(args[19], 'ONE_OR_MORE')
     #check bidder code
-    self.assertEqual(args[8], None)
+    self.assertEqual(args[8], ['mypartner'])
```

Comment:
You can update the same method for custom targeting as well. In this test case currently, custom targeting is not passed and set.